.golangci.yml

```diff
@@ -44,6 +44,25 @@ issues:
 
 
 linters-settings:
+  errcheck:
+    check-type-assertions: false
+    check-blank: false
+    disable-default-exclusions: false
+    exclude-functions:
+    - (*bytes.Buffer).WriteString
+    - (*strings.Builder).WriteByte
+    - (*strings.Builder).WriteRune
+    - (*strings.Builder).WriteString
+    - fmt.Fprint
+    - fmt.Fprintf
+    - fmt.Fprintln
+    - fmt.Print
+    - fmt.Printf
+    - fmt.Println
+    - github.com/stackrox/rox/pkg/utils.Should
+    - io.Copy(*bytes.Buffer)
+    - io.Copy(os.Stdout)
+    - io/ioutil.ReadFile
```

Comment:
We already have a rule which disallows `io/ioutil`. Should this be replaced with `os.ReadFile`? Also, when would someone want to ignore the output of this anyway?